xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/pom.xml

```diff
@@ -48,5 +48,18 @@
       <artifactId>xwiki-platform-like-notifications</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <!-- Runtime dependencies. -->
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-livedata-macro</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-livedata-livetable</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
why don't you add an optional dep on the user profile since we add a new tab for likes?

Reply:
> what options do you have when you click the kebab menu?

![Screenshot_2021-05-03 Profile of Administrator - XWiki(3)](https://user-images.githubusercontent.com/327856/116885683-736b1900-ac28-11eb-8c05-1eb0bcea129f.png)

Reply:
ah so we can filter and sort now? I was surprised to not see any filtering inputs in the view UI as we had for LT. Is that a choice we have now? Marius explained that to me but I don't remember ;)

Reply:
Filtering and sorting are supported by LD, it just happens that the migrated LT did not support them.

Reply:
> Filtering and sorting are supported by LD,

AFAIR we support inputs for simple search and the kebab menu for advanced filtering, but not sure.

Reply:
Yes, if a column can be filtered a filter header is displayed, allowing to filter on a single criteria. Using the filter menu from the kebab button allows for advanced filter, not sure how advanced though.

Reply:
> not sure how advanced though.

You can combine several filters for one column with AND or OR AFAIR.

Reply:
@vmassol the (old) "Liked Pages" live **table** was neither filterable nor sortable so you shouldn't be surprised that the (new) live **data** is neither filterable nor sortable. Both use the same source (which doesn't support sort and filter ATM).

@manuelleduc BTW, when migrating live tables, please compare the result you get after your changes (with the live data macro) with the result you get before your changes but with ``?useLiveData=true`` in the URL. It should be the same.